src/views/PositionPlot/PositionPlotViewData.ts

```diff
@@ -0,0 +1,18 @@
+import { validateObject } from "figurl"
+import { isArrayOf, isEqualTo, isString } from "figurl/viewInterface/validateObject"
+
+export type PositionPlotViewData = {
+    type: 'PositionPlot'
+    timestamps: number[]
+    positions: number[][]
+    dimensionLabels: string[]
+}
+
+export const isPositionPlotViewData = (x: any): x is PositionPlotViewData => {
+    return validateObject(x, {
+        type: isEqualTo('PositionPlot'),
+        timestamps: () => (true),
+        positions: () => (true),
```

Comment:
Assuming one mark of invalid data would be if these two lists have different lengths.